sky/usage/usage_lib.py

```diff
@@ -143,8 +143,10 @@ def __init__(self) -> None:
         #: Requested number of nodes
         self.task_num_nodes: Optional[int] = None  # update_actual_task
         # YAMLs converted to JSON.
-        self.user_task_yaml: Optional[str] = None  # update_user_task_yaml
-        self.actual_task: Optional[Dict[str, Any]] = None  # update_actual_task
+        self.user_task_yaml: Optional[List[Dict[
```

Comment:
Is this typing right?

Reply:
Ahh, actually the `usage_lib` may have to be fixed for the pipeline, as we currently assume the dag is always a single task.